electron/electron.js

```diff
@@ -74,3 +74,25 @@ ipcMain.handle('get-file', async () =>
     properties: ['openFile'],
   }),
 );
+
+ipcMain.handle('get-output-path', async () => {
+  const options = {
+    title: 'Select Output Folder',
+    defaultPath: app.getPath('downloads'),
+    properties: ['openDirectory', 'createDirectory'],
+  };
+  return dialog.showOpenDialog(null, options, (savePath) => savePath);
+});
+
+ipcMain.handle('save-output', async (event, savePath, extractedData) => {
+  try {
+    extractedData.forEach((bundle, i) => {
+      const outputFile = path.join(savePath, `mcode-extraction-patient-${i + 1}.json`);
+      fs.writeFileSync(outputFile, JSON.stringify(bundle), 'utf8');
+      // 'false' as in 'no error'
```

Comment:
DP2 - This placement and text feels outdated. Can we update this comment to be closer to what it's referring to and to be more accurate?

Reply:
@SenecaAnderson to be clear, I didn't ask you to remove the comment. Could we add an updated comment that sensibly communicates what false and true mean in this context?